README.md

```diff
@@ -103,7 +103,13 @@ Pretty easy. Just check out the code, then run:
 $ npm install
 ```
 
-That'll pull down the dependencies. Then run:
+That'll pull down the dependencies. 
+
+```bash
+$ bower install
```

Comment:
Hmm, I see it there - but I ran bower install, not the version installed in gulp - so maybe it was a bower version mismatch on my machine or something? I'll remove it til we see if other folks have problems.